src/utils/Chart.js

```diff
@@ -0,0 +1,35 @@
+const d3 = require('d3');
+
+const Chart = {
+  getDataMinMaxValues (data, axis) {
+    const max = d3.max(data, d => {
+      return Math.ceil(d[axis] / 1000) * 1000;
```

Comment:
It is not.  The dividing by and then multiplying by a thousand is a way to give our selves a cushion for the min and max values.  Other wise the data points end up being to close to the top and bottom of the graph.  This ensures that the data ends up being more centralized in the graph.